tests/test_plugins.py

```diff
@@ -64,6 +66,9 @@ def configure_worker(self, config: WorkerConfig) -> WorkerConfig:
 class MyWorkerPlugin(temporalio.worker.Plugin):
     def configure_worker(self, config: WorkerConfig) -> WorkerConfig:
         config["task_queue"] = "replaced_queue"
+        runner = config.get("workflow_runner")
+        if isinstance(runner, SandboxedWorkflowRunner):
+            runner.restrictions.passthrough_modules.add("my_module")
```

Comment:
Restrictions are not usually meant to be mutated. Doing this in a plugin will, for many, mutate the shared default sandbox parameter for everyone. Usually I would expect this would be something like:

```python
if isinstance(runner, SandboxedWorkflowRunner):
    config["runner"] = dataclasses.replace(
        runner,
        restrictions=runner.restrictions.with_passthrough_modules("my_module")
    )
```

But we can't really change restriction sets to be frozen sets now. But we should probably `frozen=True` the sandbox (which means if you make those workflow exception types a field, you'll have to cheat the frozen aspect to set the attr, I can show how if needed).